libtextsecure/outgoing_message.js

```diff
@@ -362,20 +389,46 @@ OutgoingMessage.prototype = {
           sourceDevice: 1,
           destinationRegistrationId: ciphertext.registrationId,
           content: ciphertext.body,
+          number: devicePubKey,
```

Comment:
Do we want to just name this pubKey or whatever if it is something we are adding?

Reply:
Sounds good!